xla/service/latency_hiding_scheduler_test.cc

```diff
@@ -3074,6 +3088,51 @@ ENTRY AddR2 {
   XLA_VLOG_LINES(1, hlo_module->ToString());
 }
 
+// Function to retrieve the value for a specific key from HloPassMetadata
+// for a given pass_name. Returns 0 if the key or pass_name is not found.
+int64_t GetKeyValueFromHloPassMetadata(const HloModuleMetadataProto& metadata,
+                                       const std::string& pass_name,
+                                       const std::string& target_key) {
```

Comment:
Prefer absl::string_view:
```suggestion
int64_t GetKeyValueFromHloPassMetadata(const HloModuleMetadataProto& metadata,
                                       absl::string_view pass_name,
                                       absl::string_view target_key) {
```

Reply:
Done.